packages/hash/frontend/src/components/layout/PageHeader/AccountDropdown.tsx

```diff
@@ -25,19 +24,27 @@ export const AccountDropdown: VoidFunctionComponent<AccountDropdownProps> = ({
   logout,
   user,
 }) => {
-  const theme = useTheme();
-
   const buttonRef = useRef(null);
 
   const [open, setOpen] = useState(false);
 
   const id = open ? "account-popover" : undefined;
 
+  const userEmail = useMemo(() => {
```

Comment:
I'm noticing the double scroll-bar issue we had on the BP site:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/42802102/158066064-13c7244c-a11b-4d37-9d1b-46e406334d18.png">